docs/js/online.js

```diff
@@ -165,12 +165,13 @@ $(document).ready(function() {
           newInput.length,
           timeAfter - timeBefore
         ));
-      $("#output").removeClass("disabled").text(util.inspect(output, {
+      $("#output").removeClass("disabled").html(util.inspect(output, {
         depth: Infinity,
         color: false,
         maxArrayLength: Infinity,
         maxStringLength: Infinity,
         compact: false,
+        stylize: util.stylizeWithHTML,
```

Comment:
This is the place where coloration was added.

Reply:
and now I need to file a bug with node-inspect-extracted to HTML-escape output in stylizeWithHTML.